### DISCUSSION TITLE
12 hour time

### DISCUSSION THREAD

**Discussion by jerbear2008** [maximmax42/Discord-CustomRP#89] [Ideas]
Can you make the local time timestamp support 12 hour time with a checkbox to toggle between that and 24 hour time?

**Comment by maximmax42** [comment]
I can try, not sure where to put AM/PM indication tho.

**Reply by jerbear2008** [reply]
I'd think a checkbox next to the radio button would be a good spot

**Reply by maximmax42** [reply]
No I mean, in the presence itself. I can't change "elapsed" or "remaining", so I'd have to put it somewhere else.

**Reply by jerbear2008** [reply]
Oh, I see. Maybe in the row above? I don't know.
